tools/scripts/run_broker_publisher.sh

```diff
@@ -200,14 +200,17 @@ case ${testplan_user_inputs["MessageSize"]} in
 	10KB)
 		text_message_file_location="sample_messages/10kB.json"
 		;;
+    40KB)
+		text_message_file_location="sample_messages/40kB.json"
+		;;
 	100KB)
         text_message_file_location="sample_messages/100kB.json"
 		;;
 	1MB)
 		text_message_file_location="sample_messages/1MB.json"
 		;;
 	?)
-	    echo message_size parameter should be one of 1KB,10KB,100KB and 1MB
+	    echo message_size parameter should be one of 1KB,10KB,40KB,100KB and 1MB
```

Comment:
There should be a space after each comma

Reply:
Done